src/server-worker.ts

```diff
@@ -0,0 +1,7 @@
+import app from "./core/index";
+import { uuid } from "@cfworker/uuid";
+import { KVWorker } from "./worker/db";
+
+globalThis.tokenGen = uuid;
```

Comment:
It looks like Cloudflare Workers also supports `crypto.randomUUID`: https://developers.cloudflare.com/workers/runtime-apis/web-crypto/#methods

Reply:
Oh, It is nice! Thank you!